main_f4.c

```diff
@@ -331,6 +416,19 @@ main(void)
 	/* do board-specific initialisation */
 	board_init();
 
+	/* configure the clock for bootloader activity */
+	rcc_clock_setup_hse_3v3(&clock_setup);
+
+	/* start the interface */
+#if INTERFACE_USART
+	cinit(BOARD_INTERFACE_CONFIG_USART, USART);
```

Comment:
The interfaces also now have to be setup earlier in the code, so that if the USART is compiled in, I can check if a LIN break was received.